.editorconfig

```diff
@@ -0,0 +1,15 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
```

Comment:
Will this incorrectly indent ml files which use ocp-indent?

Reply:
I can copy our `.ocp-indent` file too